pkg/apis/build/v1beta1/build_conversion.go

```diff
@@ -284,10 +284,12 @@ func (p ParamValue) convertToAlpha(dest *v1alpha1.ParamValue) {
 	}
 
 	if p.ConfigMapValue != nil {
-		dest.ConfigMapValue = &v1alpha1.ObjectKeyRef{}
+		dest.SingleValue = &v1alpha1.SingleValue{}
 		dest.ConfigMapValue = (*v1alpha1.ObjectKeyRef)(p.ConfigMapValue)
```

Comment:
This took me a while to understand. Can you change this and the similar occurences in this file to:

```suggestion
		dest.SingleValue = &v1alpha1.SingleValue{
			ConfigMapValue: (*v1alpha1.ObjectKeyRef)(p.ConfigMapValue),
		}
```